src/Microsoft.CodeAnalysis.Analyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -0,0 +1,6 @@
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+RS1031 | MicrosoftCodeAnalysisDesign | Warning | DiagnosticDescriptorCreationAnalyzer
```

Comment:
Can you clear out this file? This should in turn prompt a code fix for you to generate the entries.

Reply:
It doesn't. I have also tried to run a `git clean -dfx` to force cleaning the local repo but no analyzer is showing for the rule. I am doing a repair of my VS but I am wondering if the last update did broke the analyzer.

Reply:
I have reverted VS and it seems to be ok now.